assets/js/lib/test-utils/factories/users.js

```diff
@@ -0,0 +1,21 @@
+import { faker } from '@faker-js/faker';
+import { Factory } from 'fishery';
+
+export const userFactory = Factory.define(() => ({
+  id: faker.number.int(),
+  username: faker.internet.userName(),
+  created_at: faker.date.past().toISOString(),
+  actions: 'Delete',
+  enabled: faker.datatype.boolean(),
+  fullname: faker.internet.displayName(),
+  email: faker.internet.email(),
+}));
+
+export const adminUser = userFactory.params({
+  id: 1,
+  username: 'admin',
+  created_at: faker.date.past().toISOString(),
```

Comment:
Do you need to specify this again? it also does `faker.date.past().toISOString()` like the default `userFactory`, right?

Reply:
I will go with the formatISO thingy we use at other parts of the app. 
```import { formatISO } from 'date-fns';
.....
  created_at: formatISO(faker.date.past()),
.....
```